### ISSUE TITLE
Function args hide global variable `r_dlightframecount`

### ISSUE THREAD

**Issue by BraXi** [yquake2/yquake2#1093]
Some functions hide global variable `r_dlightframecount` by using argument with the same name

```C
void GL3_MarkSurfaceLights(dlight_t *light, int bit, mnode_t *node, int r_dlightframecount)
void R_MarkSurfaceLights(dlight_t *light, int bit, mnode_t *node, int r_dlightframecount)
void R_MarkLights (dlight_t *light, int bit, mnode_t *node, int r_dlightframecount, marksurfacelights_t mark_surface_lights);
```

**Comment by 0lvin** [comment]
Yes, it's bad code style. Shared code does not use global variables and receive all such values as parameter.Better to rename parameter name to something else, are you prefer to create pull request to yquake repository, or I will create pull request with fix and mark you as author of fix?

I will create fix for remaster in any case.

**Comment by BraXi** [comment]
The change is trival and since you offer to PR yourself its okay

**Comment by Yamagi** [comment]
Fixed by merging #1094.
